### PR TITLE
Feature/ Added Pull to refresh.

### PR DESCRIPTION
**Pull request by Mekhak11**

### CODE REVIEW

File: Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      HStack {
+        Spacer()
+        if needRefresh {
+          ProgressView()
+        } else {
+          Text("🎂")
+        }
+        Spacer()
+      }
+    }.padding(.top, -50)
```
Comment (Aslanyan99n) [inline Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift]: Please add modifier in new line

File: Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//  Created by MEKHAK GHAPANTSYAN on 28.10.24.
+//
+
+import Foundation
+import SwiftUI
+
+struct PullToRefresh: View {
+  
+  var coordinateSpaceName: String
+  var onRefresh: ()->Void
```
Comment (Aslanyan99n) [inline Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift]: add spaces for closure

File: Birthday/Birthday/Modules/Home/HomePage/View/BirthDayCell.swift [hunk 3]
```diff
@@ -31,19 +31,29 @@ extension BirthDayCell {
     .cornerRadius(24)
   }
   
+  @ViewBuilder
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Modules/Home/HomePage/View/BirthDayCell.swift]: is view builder needed here?

File: Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift (hunk 1 above)
Comment (Mekhak11) [reply Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift]: fixed

File: Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift (hunk 2 above)
Comment (Mekhak11) [reply Birthday/Birthday/Utils/PullToRefresh/PullToRefresh.swift]: fixed.

File: Birthday/Birthday/Modules/Home/HomePage/View/BirthDayCell.swift (hunk 3 above)
Comment (Mekhak11) [reply Birthday/Birthday/Modules/Home/HomePage/View/BirthDayCell.swift]: fixed

**Review by Aslanyan99n — APPROVED** [review]

**Review by nellyAslanyan — APPROVED** [review]

**Pull request merged**